### PR TITLE
Bocaj Ergodox Updates

### PR DESCRIPTION
**Pull request by JacobJerrell**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,6 +37,7 @@ doxygen/
 /keyboards/*/*/*/keymaps/Makefile
 /keyboards/*/*/*/*/keymaps/Makefile
 /keyboards/*/*/*/*/*/keymaps/Makefile
+secrets.h
```
Comment (drashna) [inline .gitignore]: This should be in ` .git/info/exclude`, not here, as this file is global, and ` .git/info/exclude` is local only.

Comment (JacobJerrell) [reply .gitignore]: Removed secrets.h - Macros have all been removed. I missed this line during the cleanup. Thank you!

File: keyboards/ergodox_ez/keymaps/bocaj/keymap.c [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    XXXXXXX, KC_1  , KC_2  , KC_3  , KC_4  , KC_S  ,                  XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,
+    KC_LSFT, KC_Z  , KC_G  , KC_O  , KC_Y  , KC_F  ,XXXXXXX,  XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,
+    XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,                                  XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,XXXXXXX,
+                                           XXXXXXX,XXXXXXX,    XXXXXXX,XXXXXXX,
+                                                   XXXXXXX,    XXXXXXX,
+                                   KC_MS_L,KC_MS_R,TO(_HWRKMN),    XXXXXXX,XXXXXXX,XXXXXXX
+  )
+};
+
+const keypos_t hand_swap_config[MATRIX_ROWS][MATRIX_COLS] = {
```
[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/ergodox_ez/keymaps/bocaj/keymap.c]: @JacobJerrell was this needed here?

**Comment by drashna** [comment]
Thanks!

**Pull request merged**